source/AI.cpp

```diff
@@ -445,7 +448,11 @@ void AI::UpdateEvents(const list<ShipEvent> &events)
 		const auto &actorGovernment = event.ActorGovernment();
 		if(actorGovernment)
 		{
+			// This removes the SCANNING ship event.
+			if(event.Type() & ShipEvent::STOPPED_SCANNING)
+				governmentActions[actorGovernment][target] &= ~ShipEvent::SCANNING + 1;
```

Comment:
I'm confused, why a +1? all good if it works but I'm just finding it odd
did I just misunderstand boolean logic? that wouldnt be surprising

Whatever the reason, plz add it to the comment above

Reply:
It's not boolean logic it's a complement.  When I had it in a debugger the complement was -1025.  When doing a complement you need to add 1 to make it -1024.  Though it has been a long since I took a CS course on that I also vaguely remember twos complement doing something similar.  I'll need to look it up.